libs/providers/go-feature-flag-web/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@openfeature/go-feature-flag-web-provider",
   "version": "0.1.3",
-  "type": "commonjs",
```

Comment:
This was the primary issue. I've removed it from all packages.

The latest NX automation seems to work great if we don't interfere with it in this way.